app/models/ext_management_system.rb

```diff
@@ -400,14 +400,15 @@ def last_refresh_status
     end
   end
 
-  def refresh_ems
+  def refresh_ems(opts = nil)
```

Comment:
Is there a concern that you might explicitly send the argument `nil` to this method? If not, `opts = {}` should suffice (without the following line)